Makefile

```diff
@@ -93,8 +93,11 @@ test/go: COVERAGE ?= false
 test/go: COVERAGE_OPTS ?= -covermode=atomic
 test/go: COVERAGE_OUTPUT ?= coverage.out
 test/go: setup-envtest
-test/go: ENVTEST_BIN ?= ${PWD}/.dev/bin # We need an absolute path for setup-envtest
-test/go: KUBEBUILDER_ASSETS ?= "$(shell setup-envtest use --bin-dir $(ENVTEST_BIN) -p path)"
+# Where to find the setup-envtest binary
+test/go: GOBIN ?= ${PWD}/.dev/bin
+# We need an absolute path for setup-envtest
+test/go: ENVTEST_BIN ?= ${PWD}/.dev/bin
```

Comment:
If we put the comments on the same line with the variable definition, the variable has trailing space with its value 😇